sdk.js

```diff
@@ -308,18 +308,20 @@ class PaddleSDK {
 	 * @param {number} subscriptionID
 	 * @param {number} planID
 	 * @param {boolean} prorate
+	 * @param {boolean} billImmediately
```

Comment:
Do you mind updating both these params as optional and add their default values in here?